### PR TITLE
DOC-2387: Add live demo to AI Assistant plugin page

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-2387

Site: [Staging branch](http://docs-feature-71-doc-2387.staging.tiny.cloud/docs/tinymce/latest/ai#interactive-example)

IMPORTANT: Check Jira ticket comments for conversation about infosec of exposing openai proxy url and token.

Changes:
* DOC-2387: Add live demo to AI Assistant plugin page

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [-] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [x] Files has been included where required `(if applicable)`
- [x] Files removed have been deleted, not just excluded from the build `(if applicable)`
- [-] Files added for `New product features`, and included a `release note` entry.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by danoaky-tiny — APPROVED** [review]

**Review by MitchC1999 — APPROVED** [review]
Nice job, works perfectly!

**Review by kemister85 — CHANGES_REQUESTED** [review]
Waiting on feedback

**Review by tiny-ben-tran — APPROVED** [review]

File: antora.yml [hunk 1]
```diff
@@ -14,6 +14,8 @@ asciidoc:
     tinydrive_live_demo_url: https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinydrive/7/tinydrive.min.js
     webcomponent_url: https://cdn.jsdelivr.net/npm/@tinymce/tinymce-webcomponent@2/dist/tinymce-webcomponent.min.js
     jquery_url: https://cdn.jsdelivr.net/npm/@tinymce/tinymce-jquery@2/dist/tinymce-jquery.min.js
+    openai_proxy_url: https://openai.ai-demo-proxy.tiny.cloud/v1/chat/completions
+    openai_proxy_token: eyJhbGciOiJFUzM4NCJ9.eyJhdWQiOlsiaHR0cHM6Ly9vcGVuYWkuYWktZGVtby1wcm94eS50aW55LmNsb3VkLyJdLCJleHAiOjE3MTk3NTYwMDAsImh0dHBzOi8vb3BlbmFpLmFpLWRlbW8tcHJveHkudGlueS5jbG91ZC9yb2xlIjoicHVibGljLWRlbW8iLCJpc3MiOiJodHRwczovL2FpLWRlbW8tcHJveHkudGlueS5jbG91ZC8iLCJqdGkiOiJlYWU5MjBhMy05ZGQ0LTRkZjUtYTM4Yy04ODY5ZTJkZGIwYTQiLCJzdWIiOiJhaS1hc3Npc3RhbnQtZGVtbyJ9.hX44J5hPOCLOidK8mBAcnlAVh-ae4nVTq3W92xRLuGiF_4mGZWn1W3Ihgd7unlffz7GDayGxD_3hoQx8f64bTJg5hyHjwjKH1AZj1EKRal-NCNbqmoUa0TOm6_VXpRl2
```
[the remark this answers is not in the document]
Comment (MitchC1999) [reply antora.yml:18]: This token is publicly available on the website demos. It is not an OpenAI api key.

File: modules/ROOT/examples/live-demos/ai/index.html
Comment (kemister85) [inline modules/ROOT/examples/live-demos/ai/index.html:1]: Update the html to match the existing `source` from https://www.tiny.cloud/tinymce/features/ai-integration/

Comment (MitchC1999) [reply modules/ROOT/examples/live-demos/ai/index.html:1]: Docs has its own content. Is there a reason we need to use the website content?

**Comment by FarzadHayat** [comment]
IMPORTANT: Check Jira ticket comments for conversation about infosec of exposing openai proxy url and token. DOC-2387

**Pull request merged**